inst/@sym/sym.m

```diff
@@ -275,6 +292,24 @@
     return
   end
 
+  if (isa (x, 'function_handle'))
+    assert (nargin == 1)
+    %% Need argnames of the handle.  TODO: can do better than regex?
+    T = regexp (func2str (x), '^\@\((?:(\w+)(?:, ))*(\w+)?\).*', 'tokens');
```

Comment:
@mtmiller do you know of a better way to get the argnames of a `function_handle`?